apps/bookmark/test/build/integration/bookmark_test.js

```diff
@@ -3,7 +3,7 @@
 /* jshint node: true, mocha: true */
 /* global suiteSetup */
 
-var helper = require('helper');
+var helper = require('../../../../../build/test/integration/helper.js');
```

Comment:
Let's replace that with NODE_PATH=build/test/integration, so that we can get rid of modifying a bunch of files.